src/biblenotify/core/notifications.py

```diff
@@ -76,6 +77,7 @@ def loadVerses(self) -> dict:
 
         verses = json.loads(verses_string)
         # Choose a random verse
+        global verse
```

Comment:
Can remove this. See my comments in the Element chat for how to avoid using the global variable.

Reply:
Ok